exporter/loadbalancingexporter/resolver_dns.go

```diff
@@ -49,13 +51,14 @@ type dnsResolver struct {
 	updateLock         sync.Mutex
 	shutdownWg         sync.WaitGroup
 	changeCallbackLock sync.RWMutex
+	telemetry          *metadata.TelemetryBuilder
 }
 
 type netResolver interface {
 	LookupIPAddr(ctx context.Context, host string) ([]net.IPAddr, error)
 }
 
-func newDNSResolver(logger *zap.Logger, hostname string, port string, interval time.Duration, timeout time.Duration) (*dnsResolver, error) {
+func newDNSResolver(logger *zap.Logger, hostname string, port string, interval time.Duration, timeout time.Duration, tb *metadata.TelemetryBuilder) (*dnsResolver, error) {
```

Comment:
Nit: Other `new...Resolver` methods were updated to have each argument on a new line, we could update this as well to be consistent. Feel free to drop, not a big deal at all.